lib/plugins/repository.js

```diff
@@ -47,7 +47,7 @@ module.exports = class Repository {
     this.installationId = installationId
     this.github = github
     this.settings = Object.assign({ mediaType: { previews: ['nebula-preview'] } }, settings, repo)
-    this.topics = settings && settings.topics && typeof settings.topics === 'string' ? this.settings.topics.split(',').map(t => t.trim()) : []
+    this.topics = this.settings.topics
```

Comment:
Should we ensure that `this.settings.topics` is really an array?

Reply:
I am not sure how to handle it if it is not an array. If it is a string, I can convert it to an array, but I have to contend with the situation where if the org and suborg level settings had topics as arrays, DeepMerge would have basically tried to compare a string with array and would have overwritten the Org and suborg level topics with just this string. This means we have to handle this in DeepMerge also.

Since it was not clear how to handle this, I just reverted it back to the way it was before.